src/worker.py

```
@@ -1296,6 +1315,131 @@ async def _dispatch(request, env):
+# ---------------------------------------------------------------------------
+
+async def api_get_comments(_req, env, activity_id: str, enc_key: str):
+    """GET /api/activities/:id/comments — list comments for an activity."""
+    # Check activity exists
+    act = await env.DB.prepare(
+        "SELECT id, host_id FROM activities WHERE id = ?"
+    ).bind(activity_id).first()
+    if not act:
+        return err("Activity not found", 404)
+    # Check user is enrolled or is host
+    if act["host_id"] != user["id"]:
```

Ruff (0.15.7)

[error] Undefined name `user`

(F821)